### ISSUE TITLE
Change accessible buffer command navigation keybinding for screen reader users

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#26262]
From a screen reader user's perspective, the terminal accessible buffer is an editor and should behave as such. 

`Ctrl/Cmd+Up/DownArrow` should jump to the top and bottom line.
